examples/restaurant_demo/README.md

```diff
@@ -1,32 +1,28 @@
 ## Restaurant Demo
 
-PyDP library version for Google's Java Differential Privacy Library example - 
-https://github.com/google/differential-privacy/tree/master/examples/java.
+PyDP library version for Google's Java Differential Privacy Library example can be found [here](https://github.com/google/differential-privacy/tree/master/examples/java).
 
 Imagine a fictional restaurant owner named Alice who would like to share
 business statistics with her visitors. Alice knows when visitors enter the
 restaurant and how much time and money they spend there. To ensure that
 visitors' privacy is preserved, Alice decides to use the Differential Privacy
 library in this case PyDP library.
 
-Alice wants to share the information with potential clients which include 3 main scenarios in total.
+Alice wants to share the information with potential clients which include 4 main scenarios in total.
 
-```
-Count visits by an hour of the day: Count how many visitors enter the restaurant at every hour of a particular day.
-Count visits by day of the week: Count how many visitors enter the restaurant each day in a week. This includes two cases:
-Sum-up revenue per day of the week: Calculate the sum of the restaurant revenue per weekday.
-Sum-up revenue per day of the week with preaggregation
-```
-    
-Both the Non-private and the Private values for the above cases are calculated down below along with the graphical representation of the results and their comparisons.
 
-Notebook Implementation for the same can be found https://github.com/OpenMined/PyDP/blob/dev/examples/Tutorial_2-restaurant_demo/restaurant_demo.ipynb
+#### 1) Count visits by an hour of the day: Count how many visitors enter the restaurant at every hour of a particular day.
+#### 2) Count visits by day of the week: Count how many visitors enter the restaurant each day in a week. This includes two cases:
+#### 3) Sum-up revenue per day of the week: Calculate the sum of the restaurant revenue per weekday.
+#### 4) Sum-up revenue per day of the week with preaggregation
+
+    
+Notebook Implementation for the same can be found [here](https://github.com/OpenMined/PyDP/blob/dev/examples/Tutorial_2-restaurant_demo/restaurant_demo.ipynb)
 
-## Installation
+## To Run the Demo
 To install PyDP, use the PyPI package manager:
 
-pip install python-dp
-(If you have pip3 separately for Python 3.x, use pip3 install python-dp.)
+`pip install python-dp` (If you have pip3 separately for Python 3.x, use pip3 install python-dp.)
```

Comment:
Pls add the below two-line (apologies I should have noticed this before!) 

or 
`pip3 install python-dp`